src/libs/blueprint/conduit_blueprint_mesh_examples.cpp

```diff
@@ -2718,6 +2802,10 @@ basic(const std::string &mesh_type,
     {
         // error, not enough points to create the topo
         CONDUIT_ERROR("blueprint::mesh::examples::basic requires: " << std::endl <<
+                      "For 1D only topologies"
+                      " ( mesh_type={\"1Duniform\" or \"1Dstructured\"} )"
+                      " npts_x > 1 and npts_y == 0 and npts_z == 0"
+                      << std::endl <<
```

Comment:
Can we just support with `mesh_type="uniform"` with 1D appropriate args, instead of a new type `1Duniform`?

Reply:
I would like to.  I did it this way because `mesh_types_dims[mesh_type_index]` specifies the dimensionality for `mesh_types[mesh_type_index]`.  If we were to do away with `1Duniform` as a mesh type, I would make a new variable `int dims = mesh_types_dims[mesh_type_index]` and then set `dims = 1` if `npts_y == 0 && npts_x == 0`.

Reply:
I think it's worth doing that refactor.